client/src/app/services/open-services/open-services.component.ts

```diff
@@ -1,66 +1,67 @@
 import { Component, OnInit } from '@angular/core';
-import { TemplateService, ToasterService, ServiceService, DEFAULT_WEEK_SWITCH } from '../../_services';
-import { Template, Service } from '../../_models/index';
-import { FormArray, FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { DEFAULT_WEEK_SWITCH, ServiceService, TemplateService, ToasterService } from '../../_services';
+import { Service, Template } from '../../_models/index';
+import { AbstractControl, FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 import * as moment from 'moment';
+// tslint:disable-next-line:no-duplicate-imports
 import { Moment } from 'moment';
 import { Router } from '@angular/router';
 
 @Component({
     selector: 'app-open-services',
-    templateUrl: './open-services.component.html'
+    templateUrl: './open-services.component.html',
 })
 
 export class OpenServicesComponent implements OnInit {
 
-    templates: Template[] = new Array<Template>();
-    services: Service[] = new Array<Service>();
+    templates: Array<Template> = new Array<Template>();
+    services: Array<Service> = new Array<Service>();
     selectedTemplate: Template;
 
     templateSelectorFormGroup: FormGroup;
     servicesFormArray: FormArray;
     generalFormArray: FormArray;
     generalFormGroup: FormGroup;
 
-    constructor (private templateService: TemplateService,
-        private toasterService: ToasterService,
-        private serviceService: ServiceService,
-        private formBuilder: FormBuilder,
-        private router: Router) {
+    constructor(private templateService: TemplateService,
+                private toasterService: ToasterService,
+                private serviceService: ServiceService,
+                private formBuilder: FormBuilder,
+                private router: Router) {
         this.createForms();
     }
 
-    createForms() {
+    createForms(): void {
         this.templateSelectorFormGroup = this.formBuilder.group({
-            templateSelectorFormControl: [''/*, Validators.required*/]
+            templateSelectorFormControl: [''/*, Validators.required*/],
         });
         this.servicesFormArray = this.formBuilder.array([]);
         this.generalFormArray = this.formBuilder.array([
             this.templateSelectorFormGroup,
-            this.servicesFormArray
+            this.servicesFormArray,
         ]);
         this.generalFormGroup = this.formBuilder.group({
-            generalFormArray: this.generalFormArray
+            generalFormArray: this.generalFormArray,
         });
     }
 
-    addServiceForm(categoryId: Number, nbMax: Number, startAt: Moment, endAt: Moment) {
+    addServiceForm(nbMax: Number, startAt: Moment, endAt: Moment): void { // TODO: categoryId: Number
```

Comment:
TODO ? `categoryId` does not exist anymore, we can delete it no ?

Reply:
+1